Userland/Libraries/LibGUI/Button.cpp

```diff
@@ -132,11 +132,11 @@ void Button::set_action(Action& action)
         set_checked(action.is_checked());
 }
 
-void Button::set_icon(RefPtr<Gfx::Bitmap>&& icon)
```

Comment:
This was a so-called rvalue reference because it can only bind to temporaries (things that appear on the right-hand side of an assignment)